src/components/uri/Visualization.tsx

```diff
@@ -411,25 +407,25 @@ export class VisualizationWrapped extends React.Component<
         );
         // keep resultSpec creation in sync with AD
         const resultSpecWithDimensions = {
-            ...resultSpec,
+            ...resultSpecSorting,
             dimensions: generateDimensions(mdObject.content, visualizationType),
         };
         const treemapDefaultSorting = isTreemap(visualizationType)
             ? {
                   sorts: getDefaultTreemapSort(afm, resultSpecWithDimensions),
               }
             : {};
-        const resultSpecWithDimensionsAndSorting = {
+        const resultSpec = {
             ...resultSpecWithDimensions,
             ...treemapDefaultSorting,
         };
 
-        const dataSource = await this.adapter.createDataSource(afmWithFilters);
+        const dataSource = await this.adapter.createDataSource(afm, JSON.stringify(afm));
```

Comment:
I believe `JSON.stringify(afm)` is not needed here, as the default does som kind of `stringify` as well.

Reply:
ok, removed